Xamarin.Forms.Core/Button.cs

```diff
@@ -52,6 +52,8 @@ public class Button : View, IFontElement, ITextElement, IBorderElement, IButtonC
 
 		public static readonly BindableProperty PaddingProperty = PaddingElement.PaddingProperty;
 
+		public static readonly BindableProperty DisabledTextColorProperty = BindableProperty.Create("DisabledTextColor", typeof(Color), typeof(Button), Color.Default);
```

Comment:
```suggestion
		public static readonly BindableProperty DisabledTextColorProperty = BindableProperty.Create(nameof(DisabledTextColor), typeof(Color), typeof(Button), Color.Default);
```

Reply:
This is indeed a more type-safe way of typing this, but I'm just following how it is already done for all the other properties, to stay consistent. :-)

Reply:
it's ok to use `nameof()` even if the other ones do not. We do not fix code that works, as it defeat history, so new and old code can differ...